### DISCUSSION TITLE
Is that codegen supports the schema in json format?

### DISCUSSION THREAD

**Discussion by dbazim** [kobylynskyi/graphql-java-codegen#972] [Q&A] [answered]
Is that codegen supports the schema in json format?

**Comment by kobylynskyi** [comment, marked as answer]
@dbazim yes, this plugin supports parsing GraphQL schema in both .graphqls and .json formats.
You can supply the path to JSON file via the following property in your `pom.xml` / `build.gradle` / `build.sbt`:
`graphqlQueryIntrospectionResultPath`.
Sample of the supported JSON schema: https://github.com/kobylynskyi/graphql-java-codegen/blob/master/src/test/resources/introspection-result/sample-introspection-query-result.jsonresult.json
